cmake/macros/macro_deal_ii_add_test.cmake

```diff
@@ -306,6 +306,15 @@ FUNCTION(DEAL_II_ADD_TEST _category _test_name _comparison_file)
       SET(_target_short ${_test_name}.${_build_lowercase}) # short target name
       SET(_run_args "$<TARGET_FILE:${_target}>") # the command to issue
 
+      #
+      # If the variable ${category_test_RUNARGS_PREFIX) is nonempty prepend
+      # it to the command line of the test:
+      #
+      if(NOT "${${_category}_${_test_name}_RUNARGS_PREFIX}" STREQUAL "")
+        SET(_run_args
+          ${${_category}_${_test_name}_RUNARGS_PREFIX} "${_run_args}")
+      ENDIF()
```

Comment:
I have decided to add a very minimal hook into the testsuite proper: If the variable `${category_test_RUNARGS_PREFIX}` is defined it is added to the command line of the test.